src/core/evaluator.js

```diff
@@ -1204,7 +1205,7 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
 
       // The xobj is parsed iff it's needed, e.g. if there is a `DO` cmd.
       var xobjs = null;
-      var xobjsCache = Object.create(null);
+      var skipNonFormXobjs = Object.create(null);
```

Comment:
Nit: Since it's XObject (note the upper-case O), how about `skipNonFormXObjs` instead?